app/views/apipie/apipies/_metadata.erb

```diff
@@ -0,0 +1,10 @@
+<% if meta[:search] %>
+  <%= heading(_('Search fields'), h_level) %>
+  <%= render(:partial => "scoped_search", :locals => { :search => meta[:search] } ) %>
+<% end %>
+
+<% other_meta = meta.except(:search) %>
+<% unless other_meta.blank? %>
+  <%= heading(t('apipie.metadata'), h_level) %>
```

Comment:
Shouldn't `t('apipie.metadata)` be rather `_('Apipie metadata')`? it seems like I18n gem that we don't use here, it seems to be used on multiple places in other view too. Or does the i18n load the translation from apipie-rails? How can we ensure the set of languages translated there is the same as those we translate? I think our own translations would make more sense.

Reply:
This translation comes from apipie. The templates are rendered from apipie-rails, so it should be translated fine.